bindings/python/CMakeLists.txt

```diff
@@ -156,10 +156,33 @@ if(APPLE)
     set(PYZINC_ARCHITECTURE universal)
 endif()
 
+# Determine the next available Python developer version for the bindings.
+if(ZINC_RELEASE)
+    set(ZINC_PYTHON_DEVELOPER_VERSION "")
+else()
+    # This command will only work if you have the pypipackagemanagement installed into the
+    # Python environment in use.  To get this package use
+    #    pip install git+https://github.com/hsorby/pypipackagemanagement.git@v0.2.0#egg=pypipackagemanagement
```

Comment:
Is this going into OpenCMISS or OpenCMISS-Bindings some time?
Is this automatically obtained by manage?

Reply:
I haven't decided on where it should go at this point, I may also add it to PyPi to make it even easier to get a hold of.

No it is not automatically obtained by manage.  It will have to be made available to the Python environment that the bindings are being built against.  I think this should be done manually by the user a la numpy.

This package is not critical to the creation of the bindings in any way.  It is really only for environments that are intending on uploading to PyPi.  If you are not intending on uploading wheels to PyPi then you will not need this package and you will not notice that you don't have it.